torchbenchmark/util/backends/jit.py

```diff
@@ -8,13 +8,20 @@ def parse_torchscript_args(args: List[str]) -> argparse.Namespace:
     parser = argparse.ArgumentParser()
     # enable ofi by default
     parser.add_argument("--no-ofi", action='store_true', help="disable optimize_for_inference")
+    parser.add_argument("--fuser", type=str, default="", choices=["fuser0", "fuser1", "fuser2", "fuser3"], help="enable fuser")
     args, unknown_args = parser.parse_known_args(args)
     return args, unknown_args
 
 @create_backend
 def torchscript(model: 'torchbenchmark.util.model.BenchmarkModel', backend_args: List[str]):
     model.jit = True
     backend_args, extra_args = parse_torchscript_args(backend_args)
+    if model.device == "cpu" and backend_args.fuser == "fuser2":
+        raise NotImplementedError("Fuser only works with GPU.")
```

Comment:
In the error message, can we have the concrete fuser names like `f"{backend_args.fuser} only works with GPU`?

Reply:
Updated, thanks